clang/bindings/python/clang/cindex.py

```diff
@@ -2022,6 +2025,14 @@ def lexical_parent(self):
 
         return self._lexical_parent
 
+    @property
+    def specialized_template(self):
+        """Return the base template that this cursor is a specialization of, if any."""
```

Comment:
I wonder if "base template" should be replaced with the Standard term "primary template".